README.md

```diff
@@ -100,335 +143,398 @@ immunopepper make_bg \
+
+*Optional parameters for cancer samples filtering*: Parameters to perform the step 2 of the cancer filtering pipeline.
+- `--sample-expr-support-cancer`: #TODO: doesn't have a default. Errors if not provided? This parameter corresponds with the sample specific filtering in the cancer pipeline. The value will correspond to the normalized expression threshold that needs to be reached by each kmer in order to be considered a kmer candidate. Kmers with an expression higher than this threshold in at least one sample will be considered as cancer candidates. #TODO: are samples provided under `--ids_cancer_samples` included in this filtering?
+- `--cohort-expr-support-cancer`: Default = None. This parameter corresponds to the expression threshold in cohort specific filtering. It indicates the normalized expression value that needs to be observed in at least `--n-samples-lim-cancer` in order to consider a kmer as a cancer candidate. Kmers with an expression higher than this threshold in at least `--n-samples-lim-cancer` samples will be considered as cancer candidates. The samples provided under `--ids_cancer_samples` will not be included in cohort filtering.
+- `--n-samples-lim-cancer`: Default = None. This parameter corresponds to the number of samples threshold in cohort specific filtering. It indicated the minimum number of cancer samples in which one should see an expression higher than `--cohort-expr-support-cancer` in order to consider the kmer as a cancer candidate. Kmers with an expression higher than `--cohort-expr-support-cancer` in at least `--n-samples-lim-cancer` samples will be considered as cancer candidates. The samples provided under `--ids_cancer_samples` will not be included in cohort filtering.
+- `--path-cancer-matrix-segm`: Default = None. Path to the cancer matrix containing segment expression from samples belonging to a cohort. The matrix will have the following dimensions: [kmers * samples]. However, it is more interesting to find novel junction kmers, so a junction expression matrix provided under `--path-cancer-matrix-edge` is the best expression proxy for that kmers. It is advised to provide only `--path-cancer-matrix-edge` and skip the inclusion of this file. If both matrices are provided, junction expression will be chosen in case there is expression information for the same kmer in both matrices. This will be the output 5 of [output, mode build](#output-files)
+- `--path-cancer-matrix-edge`: Default = None. Path to the cancer matrix containing junction expression from samples belonging to a cohort. The matrix will have the following dimensions: [kmers * samples]. This is the best expression proxy for junction kmers. It is advised to provide only this file and skip the inclusion of `--path-cancer-matrix-segm`. If both matrices are provided, junction expression will be chosen in case there is expression information for the same kmer in both matrices. This will be the ouput 6 of [output, mode build](#output-files).
```

Comment:
see above

Reply:
We should not completely discourage the use of `--path-cancer-matrix-segm`. Because in the case of non-junction kmers containing mutations, one could input this matrix as a cancer foreground

Reply:
Corrected